src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+body {
+  margin: 0;
+  font-family: Arial, sans-serif;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 10px;
+  gap: $gap;
+  width: $day-rows * $box-size + ($day-rows - 1) * $gap;
+
+  @each $day, $day-number in $days {
+    &--start-day-#{$day} &__day:first-child {
+      margin-left: $day-number * ($box-size + $gap);
+    }
+  }
+
+  @for $i from 28 through 31 {
+    &--month-length-#{$i} &__day:nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $box-size;
+    height: $box-size;
+    box-sizing: border-box;
```

Comment:
You shouldn't use `*` selector for `margin` and `padding`, because it impacts the performance. 

But what about `box-sizing`? 🤔

In case you have 30 different blocks on the page, will you write `box-sizing` to all these blocks? In a big project if somebody forgot to add `box-sizing` it can be hard to debug in the future.

So it's good to write `box-sizing` in `*` selector. Let's move it there.